src/main/java/dev/jbang/util/ConsoleInput.java

```diff
@@ -68,11 +84,9 @@ public String readLine() {
 					input = result.get(timeout, unit);
 					break;
 				} catch (ExecutionException | IOException e) {
-					e.getCause().printStackTrace();
+					Util.verboseMsg("/dev/tty is not available");
```

Comment:
It won't still make sense to print exception if in verbose mode ?

Reply:
Sure